internal/apps/dop/providers/issue/dao/issue_property.go

```diff
@@ -93,33 +91,32 @@ func (client *DBClient) GetIssueProperties(req pb.GetIssuePropertyRequest) ([]Is
 	if err := db.Order("index").Find(&propertiesProject).Error; err != nil {
 		return nil, err
 	}
-	properties = append(properties, propertiesProject...)
 
 	if req.OnlyProject {
 		return propertiesProject, nil
 	}
 	// 优先级：项目 > 企业级，当有重复的字段时，项目覆盖企业的字段；
 	properties = NameConflict(properties, propertiesProject)
-
 	return properties, nil
 }
 
-// NameConflict 重名覆盖函数用于解决自定义事项不同类型，名称相同冲突问题，将lowProperties覆盖到highProperties上。优先级：app > project > org
-func NameConflict(highProperties, lowProperties []IssueProperty) []IssueProperty {
-	overlayIndex := make([]int, 0)
-	for i, highProperty := range highProperties {
-		for _, lowProperty := range lowProperties {
-			if lowProperty.PropertyName == highProperty.PropertyName && lowProperty.ScopeType != highProperty.ScopeType {
-				overlayIndex = append(overlayIndex, i)
-			}
-		}
+// NameConflict 重名覆盖函数用于解决自定义事项名称相同冲突问题，优先级：app > project > org
+func NameConflict(properties ...[]IssueProperty) []IssueProperty {
+	propertyMap := make(map[string]IssueProperty, 0)
+	// 目前没有 app 级别，properties[0] 代表组织级别，properties[1] 代表项目级别
```

Comment:
do not use index 0, 1.
it can be unlimited.

Reply:
ok，thanks